### PR TITLE
chore: automate publishing on new Git tags

### PR DESCRIPTION
**Pull request by mathiasbynens**
See the changes to `CONTRIBUTING.md` to get an idea of how much this already simplifies things.

Issues: #6482

### CODE REVIEW

File: .github/workflows/publish-on-tag.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: npm install
+      - name: Build
+        run: npm run build
+      - name: Publish puppeteer
+        env:
+          NPM_TOKEN: ${{secrets.NPM_TOKEN_PUPPETEER}}
```
Comment (mathiasbynens) [inline .github/workflows/publish-on-tag.yml:20]: Note: I’ve already added `NPM_TOKEN_PUPPETEER` and `NPM_TOKEN_PUPPETEER_CORE` as secrets to the repo.

**Review by johanbay — APPROVED** [review]
Cool! I'm looking forward to try this out!

**Comment by mathiasbynens** [comment]
https://unicode.org/reports/tr51/#def_basic_emoji_set says:

> **ED-20. basic emoji set** — The set of emoji characters and emoji presentation sequences listed in the `emoji-sequences.txt` file [emoji-data] under the type_field `Basic_Emoji`.
> 
> - This is the set of emoji intended for general-purpose input.
> - This set excludes all instances of an emoji component, which are not intended for independent, direct input.
> - This set otherwise includes all instances of an emoji character with the property value `Emoji_Presentation = Yes` and all instances of a valid emoji presentation sequence whose base character has the property value `Emoji_Presentation = No`.

The sequence U+2757 U+FE0F is a valid presentation sequence per [`emoji-variation-sequences.txt`](https://unicode.org/Public/13.0.0/ucd/emoji/emoji-variation-sequences.txt), but since U+2757 by itself already has `Emoji_Presentation = Yes` (not `No`) it’s not included in `RGI_Emoji`.

TL;DR U+2757 is `RGI_Emoji`, but U+2757 U+FE0F is not.

References to the relevant data files follow.

-----

https://unicode.org/Public/13.0.0/ucd/emoji/emoji-data.txt

```
2757          ; Emoji                # E0.6   [1] (❗)       exclamation mark
```

-----

https://unicode.org/Public/13.0.0/ucd/emoji/emoji-variation-sequences.txt

```
2757 FE0E  ; text style;  # (5.2) HEAVY EXCLAMATION MARK SYMBOL
2757 FE0F  ; emoji style; # (5.2) HEAVY EXCLAMATION MARK SYMBOL
```

-----

https://unicode.org/Public/emoji/13.1/emoji-sequences.txt

```
2757          ; Basic_Emoji                  ; red exclamation mark                                           # E0.6   [1] (❗)
```
